src/Core/tests/DeviceTests/Stubs/LabelStub.cs

```diff
@@ -11,5 +11,7 @@ public partial class LabelStub : StubBase, ILabel
 		public string FontFamily { get; set; }
 
 		public double FontSize { get; set; }
+
+		public Thickness Padding { get; set; }
```

Comment:
The stub classes are implementations of the MAUI interfaces used for on-device testing.